.gitignore

```diff
@@ -1,4 +1,3 @@
-Cargo.lock
```

Comment:
Not sure why this was excluded from the repo. Is there a good reason  for it?

Reply:
History. It's good to move to including the locks now.

Reply:
Interesting, the advice around this seems to be changing -> https://blog.rust-lang.org/2023/08/29/committing-lockfiles.html

We do have jobs on other projects to update lockfiles, I wonder if we should have one here too